diff --git a/rdf_gen.py b/rdf_gen.py
index e560379..8e45611 100755
--- a/rdf_gen.py
+++ b/rdf_gen.py
@@ -114,6 +114,9 @@ def read_efm(filename):
 
         # now the data has 4 elements
         data = [elem.strip() for elem in data]
+        if "-" in data[2]:
+            cols = re.split('-', data[2])
+            data[2] = cols[0]
         messages.append(
             ErrorMessage(data[0], int(data[1]), int(data[2]), data[3])
         )
